src/CGRANode.cpp

```diff
@@ -425,13 +425,11 @@ bool CGRANode::enableFunctionality(string t_func) {
     string type;
     if (t_func.length() == 4) type = "none";
     else type = t_func.substr(t_func.find("call") + 5);
-    cout << type << endl;
     enableCall(type);
   } else if (t_func.find("complex") != string::npos) {
     string type;
     if (t_func.length() == 7) type = "none";
```

Comment:
ditto

Reply:
sry but I remembered deleteing these printing statements and this line has been removed from the code.